extensions/wikia/AdEngine/js/PageFairDetection.js

```diff
@@ -0,0 +1,73 @@
+define('ext.wikia.adEngine.pageFairDetection', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.utils.scriptLoader',
+	'wikia.document',
+	'wikia.log',
+	'wikia.window'
+], function (adContext, scriptLoader, doc, log, window) {
+	'use strict';
+
+	var context = null,
+		logGroup = 'ext.wikia.adEngine.pageFairDetection';
+
+	function getContext() {
+		if (context === null) {
+			context = adContext.getContext();
+		}
+		return context;
+	}
+
+	function isMobile() {
```

Comment:
isMobile means is either mobile phone or tablet? If yes please either add a comment or name a function in a more descriptive way (second one prefered)

We have module called `browserDetect` in resources/wikia/modules - it looks like a right place to put this kind of method there.